values.yaml

```diff
@@ -69,18 +69,17 @@ sts:
 
 properties:
   # use externalSecure for when inbound SSL is provided by nginx-ingress or other external mechanism
+  sensitiveKey: changeMechangeMe # Must to have minnimal 12 length key
   externalSecure: false
-  isNode: true # set to false if ldap is enabled
-  httpPort: 8080 # set to null if ldap is enabled
-  httpsPort: null # set to 9443 if ldap is enabled
-  webProxyHost:
+  isNode: flase
+  httpsPort: 8443 
+  webProxyHost: # <clusterIP>:<NodePort> (If Nifi service is NodePort or LoadBalancer)
   clusterPort: 6007
-  clusterSecure: false # set to true if ldap is enabled
+  clusterSecure: true
   needClientAuth: false
   provenanceStorage: "8 GB"
   siteToSite:
     port: 10000
-  authorizer: managed-authorizer
   # use properties.safetyValve to pass explicit 'key: value' pairs that overwrite other configuration
   safetyValve:
     #nifi.variable.registry.properties: "${NIFI_HOME}/example1.properties, ${NIFI_HOME}/example2.properties"
```

Comment:
if you want the port-forward to workout, I figured out this morning that you need to setup the following https properties under safetyValve:

```
safetyValve:
    #nifi.variable.registry.properties: "${NIFI_HOME}/example1.properties, ${NIFI_HOME}/example2.properties"
    nifi.web.http.network.interface.default: eth0
    # listen to loopback interface so "kubectl port-forward ..." works
    nifi.web.http.network.interface.lo: lo
    # Cluster Secure requirements
    nifi.web.https.network.interface.default: eth0
    # listen to loopback interface so "kubectl port-forward ..." works
    nifi.web.https.network.interface.lo: lo
```